shell/components/templates/blank.vue

```diff
@@ -9,7 +9,10 @@ export default {
 </script>
 
 <template>
-  <main class="main-layout">
+  <main
+    class="main-layout"
+    :aria-label="t('layouts.blank')"
```

Comment:
What is the benefit to adding an aria label to our `main` tags to describe the layout?

Reply:
Apparently it's a best practice, as per:

<img width="2192" alt="Screenshot 2025-03-27 at 11 52 42" src="https://github.com/user-attachments/assets/0e8ed3ac-8ff2-422c-a877-db367de33554" />

https://github.com/dequelabs/axe-core/blob/develop/doc/rule-descriptions.md (search for `landmark-unique`)
https://dequeuniversity.com/rules/axe/4.10/landmark-unique?application=RuleDescription

Reply:
This seems to apply to multiple landmarks being present on the page[^1]. Is it the case that we have multiple `main` tags rendered?

[^1]: https://www.w3.org/WAI/ARIA/apg/patterns/landmarks/examples/main.html

Reply:
Given your last link, I think you might be right. Nevertheless, the accessibility report clearly states two different points:
`landmark-no-duplicate-main`, which was the case and the PR addresses that (There's no more than 1 `main` rendered)
`landmark-unique`, which is the adding of the `aria-label`

Reply:
I think that we can keep them. I think that you're reasoning to adding them is more than sufficient and addition of the labels can provide more context for screen reader users.